control-plane-operator/controllers/hostedcontrolplane/v2/ocm/config.go

```diff
@@ -38,7 +39,7 @@ func adaptConfigMap(cpContext component.WorkloadContext, cm *corev1.ConfigMap) e
 		return fmt.Errorf("failed to read observed global config: %w", err)
 	}
 
-	adaptConfig(config, cpContext.HCP.Spec.Configuration, cpContext.ReleaseImageProvider, observedConfig.Build)
+	adaptConfig(config, cpContext.HCP.Spec.Configuration, cpContext.ReleaseImageProvider, observedConfig.Build, cpContext.HCP.Spec.Capabilities)
```

Comment:
thanks! I see you also change other components like oapi and cvo, can you please also make the change for their v2 counter parts as you did here.

Reply:
Done.